### DISCUSSION TITLE
React Native full Screen Radial/Linear Gradient

### DISCUSSION THREAD

**Discussion by jim4067** [tamagui/tamagui#2469] [Q&A]
Is there a way to direct the `LinearGradient` component to take up the full screen? 

related, is there support for radial gradient or plans to do so in the future?

**Comment by byteab** [comment]
you can use `fullScreen` prop on linear gradient, which internally uses `position: "absolute", top: 0, left: 0, right: 0, bottom: 0`
we have a PR that will add some web missing props, it should be ready some day next week

**Reply by jim4067** [reply]
Thanks for the response. I'll keep an eye out for that. 

final question, what about `radial gradients`? Is this possible? How can I implement this? I am new to react native with expo router.

**Reply by byteab** [reply]
that PR that I said will include that as well, but for native I think there might be some libraries
